cfme/middleware/__init__.py

```diff
@@ -27,6 +31,17 @@ def _on_detail_page(self):
         return sel.is_displayed('//h1[contains(., "{} (Summary)")]'.format(self.name))
 
 
+import_form = Form(
+    fields=[
+        ("file_select", "#upload_file"),
+        ("enable_deployment", "#enable_deployment_cb"),
+        ("runtime_name", "#runtime_name_input"),
```

Comment:
Oh, and the enable_deployment is actually a checkbox, that might stay is it is for now, seems we don't have a class for that yet.